assets/js/components/settings/SettingsStatuses.js

```diff
@@ -21,11 +21,30 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ProgressBar } from 'googlesitekit-components';
+
 export default function SettingsStatuses( { statuses } ) {
 	if ( ! statuses || statuses.length === 0 ) {
 		return null;
 	}
 
+	function renderStatus( status ) {
+		if ( status === undefined ) {
+			return <ProgressBar />;
+		}
+		return status
+			? __( 'Enabled', 'google-site-kit' )
+			: __( 'Disabled', 'google-site-kit' );
```

Comment:
We could do something like this:
```suggestion
		if ( status === undefined ) {
			return (
				<ProgressBar className="googlesitekit-settings-module__meta-item-data" />
			);
		}

		return (
			<p className="googlesitekit-settings-module__meta-item-data">
				{ status
					? __( 'Enabled', 'google-site-kit' )
					: __( 'Disabled', 'google-site-kit' ) }
			</p>
		);
```